src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/ModuleBuilder.cs

```diff
@@ -118,10 +118,10 @@ internal void CheckTypeNameConflict(string strTypeName, Type? enclosingType)
             return SymbolType.FormCompoundType(strFormat, baseType, 0);
         }
 
-        [DllImport(RuntimeHelpers.QCall, CharSet = CharSet.Unicode)]
+        [DllImport(RuntimeHelpers.QCall, EntryPoint = "COMModule_GetTypeRef", CharSet = CharSet.Unicode)]
```

Comment:
The `COM` prefix is legacy from the days CLR was called COM+. Could you please rename the QCall entrypoints that start with the COM prefix to use the containing managed type name so that we avoid propagating the COM+ legacy to more places?

(I have written this comment before during the original review, but it got lost somehow.)